README.md

```diff
@@ -1,44 +1,53 @@
 # JustBar
+A HMOS library to add a Bar.
 
-![](https://github.com/Hamadakram/JustBar/blob/master/art/banner.gif)
-## Download
-Grab via Gradle:
-```groove
-implementation 'com.irozon.justbar:justbar:1.0.1'
+# Source
+Inspired by [Hammad Akram/JustBar](https://github.com/Hamadakram/JustBar) - version 1.0
```

Comment:
The base lib does not have any released version yet. So we need not mention the lib version as of now.. If the base lib is released later, we can update our ReadMe likewise.

Reply:
Done